R/shinywrappers.R

```diff
@@ -781,17 +789,8 @@ downloadHandler <- function(filename, content, contentType=NA, outputArgs=list()
 #'   `c(1, 3, 4)`, or `c(-1, -3)` (all columns except the first and
 #'   third), or `c('Species', 'Sepal.Length')`.
 #' @param outputArgs A list of arguments to be passed through to the implicit
-#'   call to [dataTableOutput()] when `renderDataTable` is used
+#'   call to dataTableOutput() when `renderDataTable()` is used
```

Comment:
There should still be backticks around `dataTableOutput()`, right?

Reply:
Oops yeah.